src/realm-settings/LocalizationTab.tsx

```diff
@@ -76,31 +104,176 @@ export const LocalizationTab = ({
     defaultValue: false,
   });
 
-  const loader = async () => {
-    try {
-      const result = await adminClient.realms.getRealmLocalizationTexts({
+  const [tableKey, setTableKey] = useState(0);
+  const [max, setMax] = useState(10);
+  const [first, setFirst] = useState(0);
+  const [filter, setFilter] = useState("");
+
+  const refreshTable = () => {
+    setTableKey(tableKey + 1);
+  };
+
+  useFetch(
+    async () => {
+      const params = {
+        first,
+        max,
+      };
+      let result = await adminClient.realms.getRealmLocalizationTexts({
+        ...params,
         realm: realm.realm!,
         selectedLocale:
-          selectMenuLocale || getValues("defaultLocale") || DEFAULT_LOCALE,
+          selectMenuLocale || getValues("defaultLocale") || whoAmI.getLocale,
```

Comment:
`getLocale()` is a function, it should be executed.
```suggestion
          selectMenuLocale || getValues("defaultLocale") || whoAmI.getLocale(),
```

Reply:
Nice catch, thank you